docs/build.md

```diff
@@ -112,13 +112,6 @@ will force VP2 to use the "OpenGL -- Legacy" profile
 
 see [Autodesk docs](https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2017/ENU/Maya/files/GUID-4928A912-DA6C-4734-863B-AB5959DA73C9-htm.html)
 
-For selection to work, also need to set
-```
-MAYA_VP2_USE_VP1_SELECTION=1 
-```
-as mentioned here:
-http://help.autodesk.com/view/MAYAUL/2017/ENU/?guid=__files_GUID_343690A7_F76D_4CD2_A964_E10DA7B5BDF8_htm
```

Comment:
Unless we plan on dropping support for Maya 2018 perhaps we should we leave this in and make a note along the lines "For selection to work in versions of Maya prior to 2019, you also need to set:"

Reply:
Also, is it still necessary to use Legacy Profile or set ` MAYA_VP2_DEVICE_OVERRIDE=VirtualDeviceGL` in Maya 2019?